recipes/python-lzo/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I$PREFIX/include -I$PREFIX/include/lzo $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+$PYTHON setup.py install
```

Comment:
This seems to use `setuptools`, can you add `--single-version-externally-managed --record record.txt`? (See the example recipe for details on this.)